da/sa.py

```diff
@@ -0,0 +1,103 @@
+import time
+import logging
+import re
+from datetime import datetime
+from db import db
+from iottalkpy.dai import module_to_sa
+from config import CSM_HOST as host
+
+log = logging.getLogger("\033[1;35m[DA]: \033[0m")
+
+
+def on_register(dan):
+    print('register successfully')
+
+
+class SaClass:
+    def __init__(self, host, field, field_id, profile={}, odf_list=[],
+                 alert_range={}, device_addr=None, **kwargs):
+        self.host = host
+        self.field = field
+        self.field_id = field_id
+        self.profile = profile
+        self.odf_list = odf_list
+        self.alert_range = alert_range
+        self.device_addr = device_addr
+        self.device_name = field.name + '_DataServer'
+        self.api_url = host   # get from config CSM_HOST
+        self.device_model = 'DataServer'
+        self.push_interval = 10
+        self.interval = {'Dummy_Sensor': 3, }
```

Comment:
這個interval 裡面的值應該是空的
```suggestion
        self.interval = {}
```